src/connectedk8s/azext_connectedk8s/troubleshoot_diagnoser_job.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: azure-arc-troubleshoot-sa
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: default
+  name: secret-reader
+rules:
+- apiGroups: [""] 
+  resources: ["secrets"]
+  verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: azure-arc-troubleshoot-sa
+  namespace: default
+roleRef:
+  kind: Role 
+  name: secret-reader 
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: diagnosercontainer
+  namespace: default
+  labels:
+    app: networktest
+spec:
+  template:
+    metadata:
+      labels:
+        app: networktest
+    spec:
+      serviceAccountName: azure-arc-troubleshoot-sa
+      containers: 
+      - name: networktest1
+        image: svacr1.azurecr.io/networktest:7.0
```

Comment:
Is this a prod acr? where is the dockerfile for this job? is the build automated?

Reply:
Also, if we are hardcoding the version here, this means that any change in diagnoser container would mean that the new version of cli also has to be released? is there any reasoning behind this coupling?

Reply:
Will raise the PR for that dockerfile and script.

Reply:
@svagadia to share the PR for the container image repo as well. Setting up the build pipeline for that repo is pending. Post reviews we'll be pushing it to an MCR, this acr is a test one currently .

Reply:
Also we can avoid the version hardcoding and pull the latest always. I don't see any issues with that